examples/tree.rs

```diff
@@ -0,0 +1,61 @@
+extern crate turtle;
+
+use turtle::*;
+
+fn draw(turtle: &mut Turtle, side: f64, deep: usize) {
```

Comment:
Just like with the [dragon curve](https://github.com/sunjay/turtle/blob/master/examples/dragon.rs) example, could you add some comments about how this code works? Someone wishing to understand/modify it in the future would really benefit from that. In particular, it would be great if you could explain the meanings of these parameters and describe in some small amount of detail what this function is doing. A link to another page that explains some of the concepts may save you some writing. =)

Reply:
Question: should `deep` be named `depth`?